test/jdk/valhalla/valuetypes/ArrayElementVarHandleTest.java

```diff
@@ -24,116 +24,93 @@
 
 /*
  * @test
- * @summary test VarHandle on primitive class array
+ * @summary test VarHandle on value class array
  * @compile -XDenablePrimitiveClasses ArrayElementVarHandleTest.java
```

Comment:
It looks like the compiler does not need -XDenablePrimitiveClasses.
Except in RecursiveValueClass.java that still has a primitive class declaration.

Reply:
Updated.